libshaderc/include/shaderc.h

```diff
@@ -128,7 +128,10 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_preprocessing_only_mode(
     shaderc_compile_options_t options);
 
-// Sets the compiler mode to suppress warnings.
+// Sets the compiler mode to suppress warnings. Note this option overrides
```

Comment:
I think by changing `Note this option overrides` to `Overrides`, it could be simpler.